src/Controller/AccessDelegatorTrait.php

```diff
@@ -31,4 +33,21 @@ protected function getDelegatedAccess(Request $request): DelegatedAccess
 
         return $delegatedAccess;
     }
+
+    protected function getType(): ?string
+    {
+        if (\method_exists($this, 'getSpaceType')) {
```

Comment:
we could refactor this after for using interface instead

Reply:
Yes, this is not the same everywhere :'( ^^